### ISSUE TITLE
Support for Python 3.11 in azurerm_linux_web_app

### ISSUE THREAD

**Issue by Ondkloss** [hashicorp/terraform-provider-azurerm#20021]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Python 3.11 is available via the Azure portal for a web app, but invalid input for `azurerm_linux_web_app`, as  3.10 is the highest accepted version. Support for 3.11 should be added.

### New or Affected Resource(s)/Data Source(s)

azurerm_linux_web_app

### Potential Terraform Configuration

```hcl
resource "azurerm_linux_web_app" "wa" {
  ...

  site_config {
    application_stack {
      python_version = "3.11"
    }
  }
}
```

**Comment by Ondkloss** [comment]
I've also added a PR #20021 to address this lack of version support.

**Comment by Ondkloss** [comment]
Fixed with the merge of #20001.
